packages/base-driver/lib/basedriver/core.ts

```diff
@@ -270,13 +270,11 @@ export class DriverCore<const C extends Constraints, Settings extends StringReco
     const currentAutomationName = _.toLower(this.opts.automationName);
 
     const parseFullName = (fullName: string) => {
-      const separatorPos = fullName.indexOf(FEATURE_NAME_SEPARATOR);
+      let separatorPos = fullName.indexOf(FEATURE_NAME_SEPARATOR);
       if (separatorPos < 0) {
-        // This should not happen as we preprocess corresponding server arguments in advance
-        throw new Error(
-          `The full feature name must include both the driver name/wildcard and the feature ` +
-          `name split by a colon, got '${fullName}' instead`
-        );
+        // Workaround for lower appium packages
+        fullName = `${ALL_DRIVERS_MATCH}${FEATURE_NAME_SEPARATOR}${fullName}`;
+        separatorPos = fullName.indexOf(FEATURE_NAME_SEPARATOR);
```

Comment:
We need this workaround to keep the insecure flag work. Removing the Error is not sufficient since it doesn't work with secure flags.

This change is to minimize the code change purpose only.

I have tested this works with `appium --allow-insecure chromedriver_autodownload` on my local.